proto/minder/v1/minder.proto

```diff
@@ -1245,15 +1245,21 @@ message UserRecord {
     reserved "organization_id";
 }
 
-// list users
+message ProjectRole {
+    Role role = 1;
+    Project project = 2;
+}
+
 // get user
 message GetUserRequest {
     reserved 1; // deprecated context
 }
 
 message GetUserResponse {
     optional UserRecord user = 1;
+    // This will be deprecated in favor of the project_roles field
     repeated Project projects = 2;
```

Comment:
What if we add `[deprecated=true]` here, so that any client that tries to use this field gets a warning message?

Reply:
thanks! I'll update it now 👍

Reply:
Addressed it 👍